src/main/java/seedu/address/logic/commands/UndoCommand.java

```diff
@@ -9,12 +9,12 @@
 //@@author A0139817U
 public class UndoCommand extends TaskCommand {
     
-    public static final String COMMAND_WORD = "undo";
+    public static final String[] COMMAND_WORD = {"undo"};
```

Comment:
Oh I thought of adding another variable but because of the parserSelector class that uses the getField method means that I have to add another command word field for all other command classes also